addons/mail/static/src/new/composer/composer.js

```diff
@@ -224,7 +224,7 @@ export class Composer extends Component {
             const { id: parentId, isNote, resId, resModel } = messageToReplyTo || {};
             const postData = {
                 attachments: this.attachmentUploader.attachments,
-                isNote: this.props.composer.type === "note" || isNote,
+                isNote: this.props.composer.thread.chatter?.composerAction === "logNote" || isNote,
```

Comment:
In regard to previous comment, assuming thread has only one chatter is potentially not correct, and this is why "send message" was in "composer view" and not in "composer" in old code.